src/esp/assets/ResourceManager.cpp

```diff
@@ -83,6 +84,11 @@ using metadata::managers::StageAttributesManager;
 
 namespace assets {
 
+// used for shadertype specification
+typedef metadata::attributes::ObjectInstanceShaderType ShaderTypeEnum;
+// used to simplify access to magnum materialAttributes
+typedef Mn::Trade::MaterialAttribute MnMatAttr;
```

Comment:
Same here, can we _please_ stop making aliases that are a *completely different name*? :pleading_face: Doing `using Mn::Trade::MaterialAttribute;` would do the job as well and won't be conflicting with anything if it stays in the scope of the cpp file (or, **ideally**, in just the one or two functions that use it, since the file is several thousand lines long).

And to be frank, except for the `Mn` / `np` / `Cr` that already went canon, I don't like the word shortening happening at various places, at all, sorry. While it probably makes sense at the point of writing the code, a month from now you'd half-forget and have to double check what it means, and when someone else comes to the code they'd have no idea at all. Most IDEs and editors can autocomplete (if I write `MatA` and hit <kbd>Ctrl</kbd> <kbd>Space</kbd> it'll fill in the rest) and given that the code is written once and then read several times, the shortenings areonly barely helping you type it out while hurting readability a lot.

Reply:
Fair enough -  I'm not attached to this mechanism.  

I was mainly thinking about future maintainers (I didn't think MnMatAttr was that different from "Magnum Material Attributes" but I won't argue that point :) ) being able to find things (since ResourceManager touches on -so many things-.)